app/views/shared/_flash_banner.html.erb

```diff
@@ -10,7 +10,7 @@
     </div>
     <div class="govuk-notification-banner__content">
       <p class="govuk-notification-banner__heading">
-        <%= msg %>
+        <%= msg.html_safe %>
```

Comment:
Pretty sure this would make us fail any and all security checks...

worth having a look at what has been done for caseworker...

```<p><%= sanitize(detail, tags: %w[a]) %></p>```

the key here is that it only allow the `a` tag to be passed in (can extend the list if required)